src/components/dashboard/components/SourceCard.jsx

```diff
@@ -30,7 +30,7 @@ const SourceCard = (source) => {
           </div>
           <div className="description col-12 text-left">
             <p>
-              <a href={`/dashboard/concepts/${owner}/${name}`} className="source-type">
+              <a href={`/dashboard/concepts/${owner_type}/${owner}/${name}`} className="source-type">
```

Comment:
What criteria do you use to tell whether to use owner_type or ownerType?

Reply:
@dkayiwa owner_type is the name that was used in the API when labeling the owner type. In the instance above I could not change it.
But where I used ownerType I was following the Javascript conventions for the camel casing.

Reply:
Fair enough 😊